Nrjwolf/unity-ios-easy-native-alert
# Fix upm warnings

### Summary
Fixed Unity Package Manager warning when using the package via git url:
```
A meta data file (.meta) exists but its asset 'Packages/nrjwolf.games.iosnativealerts/IOSNativeAlert/IOS' can't be found. When moving or deleting files outside of Unity, please ensure that the corresponding .meta file is moved or deleted along with it.
```

### Details
Removed dangling .meta file which caused the warning. Made few bonus fixes after that:
* Remove macOS generated `.DS_Store` files. 132f6d5fc707e1b4fc81ff77d12176ff8df6870b
* Fixed README.md example compilation errors. 3108a2c234e8493f0044a183582f49928f256273
* Resaved package .asmdef file since Unity did not save its content on creation. To check - can modify .asmdef name and click "Apply" button. 1c6dc0b6145a754f07a21f3bae023fab6f1562de
* Extracted README.md example into package sample. Now example can be imported into project via Unity Package Manger window. Note that example code moved into `Nrjwolf.Tools` namespace not to pollute global namespace. 1200694f2e6a1c51ba03f2a1b408facfda2e0f85
* Fixed `manifest.json` path in README.md. 394baed236a594bd2f039996dfcf52fa4d94c3f9
* Renamed license file to have .md extension trying to fix MIT license not shown on the repository page, it shows "View license". 6a70bd91466efd56dd9f6fc2e2225e9f7b4f1d7d

### How to test
Install package using target branch:
```
"nrjwolf.games.iosnativealerts": "https://github.com/Nrjwolf/unity-ios-easy-native-alert.git" 
```
Observe warning mentioned in summary.

Install package using source branch:
```
"nrjwolf.games.iosnativealerts": "https://github.com/rfadeev/unity-ios-easy-native-alert.git#fix-upm-warnings" 
```
Observe no warning in the console after package import. Observe package example in package manager window. Check sample compiles for iOS platform.
